packages/AlgernonTests.package/ALGTestUI.class/instance/testAlgernonPositioningWindowIsSmallerAfterReopening.st

```diff
@@ -0,0 +1,7 @@
+testing
```

Comment:
Please also add tests for the case that the window resized but Algernon is still in view, as discussed above.